frontend/src/metabase/metadata/components/SemanticTypePicker/SemanticTypePicker.unit.spec.tsx

```diff
@@ -105,7 +103,7 @@ describe("SemanticTypePicker", () => {
     it("does not show deprecated semantic types", async () => {
       setup({ fieldId: TEMPORAL_FIELD.id });
 
-      await verifySemanticTypesVisibility({
+      await assertSemanticTypesVisibility({
```

Comment:
:x: This is a naming downgrade.
Functions named `expect*` or `assert*` suggest that they do nothing but assertions.
This particular function was intentionally called `verify*` because aside from assertions it performs some actions.

That's one of the reasons we agreed to allow more prefixes.